channeld/channeld.c

```diff
@@ -164,9 +170,15 @@ static void billboard_update(const struct peer *peer)
 
 	if (peer->funding_locked[LOCAL] && peer->funding_locked[REMOTE])
 		funding_status = "Funding transaction locked.";
-	else if (!peer->funding_locked[LOCAL] && !peer->funding_locked[REMOTE])
-		/* FIXME: Say how many blocks to go! */
-		funding_status = "Funding needs more confirmations.";
+	else if (!peer->funding_locked[LOCAL] && !peer->funding_locked[REMOTE]) {
+		if(peer->funding_locked[LOCAL])
```

Comment:
This if() will never be true

Reply:
you are right. It's a redundant if() which duplicate with the next else if()